day_two_guide/topics/default_storage_class.adoc

```diff
@@ -0,0 +1,18 @@
+////
+Checking the default storage class
+
+Module included in the following assemblies:
+
+* day_two_guide/run_once_tasks.adoc
+////
+
+For proper functionality of dynamically provisioned persistent storage, the default storage class needs to be defined. During the installation, this default storage class is defined for common cloud providers, such as AWS, GCP and more. To verify that the default storage class is defined:
+
+----
+$ oc get storageclass
+NAME                 TYPE
+ssd                  kubernetes.io/gce-pd
+standard (default)   kubernetes.io/gce-pd
+----
+
+The above output is taken from an {product-title} instance running on Google Cloud Platform, where two kinds of persistent storage are available: standard (HDD) and SSD. Notice the standard storage class is configured as the default. If there is no storage class defined, or none is set as a default, see the xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc[Dynamic Provisioning and Creating Storage Classes] section.
```

Comment:
Above 'Google Cloud Platform' was GCP - maybe make it Google Cloud Platform (GCP) above and then use GCP here.  

The same goes for AWS